.babelrc

```diff
@@ -1,19 +1,44 @@
 {
     "presets": [
-        ["es2015", {
-            "modules": false
-        }],
         "flow"
     ],
-    "plugins": [
-        "transform-runtime",
-        "transform-class-properties"
-    ],
     "env": {
-        "test": {
+        "production": {
+            "presets": [
+                ["es2015", {
+                    "modules": false
+                }]
+            ],
+            "plugins": [
+                "transform-class-properties",
+                "transform-runtime"
+            ]
+        },
+        "development": {
+            "presets": [
+                ["es2015", {
+                    "modules": false
+                }]
+            ],
             "plugins": [
-                "transform-es2015-modules-commonjs"
+                "transform-class-properties",
+                "transform-runtime"
+            ]
+        },
+        "test": {
+            "presets": [
+                "es2015"
             ],
+            "plugins": [
+                "transform-class-properties",
+                "transform-runtime"
+            ]
+        },
+        "esm": {
+            "plugins": [
+                "transform-class-properties",
+                "./contrib/babel-plugin-esm/babel-plugin-esm.js"
+            ]
```

Comment:
Duplications are inevitable due to strange `.babelrc` `env` merging: https://github.com/babel/babel/issues/5276. This can be simplified in Babel 7 using `.babelrc.js`.